webserver/forms.py

```diff
@@ -14,6 +15,16 @@
 DATASET_DONE = "done"
 DATASET_ALL = "all"
 
+DATASET_C_VALUE = ' -5, -3, -1, 1, 3, 5, 7, 9, 11 '
+DATASET_GAMMA_VALUE = '3, 1, -1, -3, -5, -7, -9, -11'
+PREPROCESSING_VALUES = [ 'basic', 'lowlevel', 'nobands', 'normalized', 'gaussianized' ]
+PREPROCESSING_VALUES = [(value, value) for value in PREPROCESSING_VALUES]
```

Comment:
This could be done in one line with a comment to clarify why we need a list of pairs here.

`PREPROCESSING_VALUES = [('basic', 'basic'), ...]`